src/Compilers/VisualBasic/Portable/Symbols/Source/ImplementsHelper.vb

```diff
@@ -351,6 +351,17 @@ DoneWithErrorReporting:
                         If Not errorReported Then
                             ' Further verification of found method.
                             foundMember = ValidateImplementedMember(implementingSym, foundMember, implementedMemberSyntax, binder, diagBag)
+
+                            If Not MembersHaveMatchingTupleNames(implementingSym, foundMember) Then
```

Comment:
>If Not MembersHaveMatchingTupleNames(implementingSym, foundMember) Then [](start = 28, length = 71)

We probably shouldn't report additional error if ValidateImplementedMember reported one. Also, it looks like ValidateImplementedMember can return Nothing, is MembersHaveMatchingTupleNames prepared to handle Nothing as an input?